packages/transformer/src/IModelTransformer.ts

```diff
@@ -329,7 +329,7 @@ function mapId64<R>(
   } else {
     throw Error(
       [
-        `Id64 container '${idContainer}' is unsupported.`,
```

Comment:
Why did you have to change this to idContainer.id ?

Reply:
I would expect that the 'isRElatedElem' check
 `const isRelatedElem = (arg: any): arg is RelatedElement =>
    arg && typeof arg === "object" && "id" in arg;` 
    
would exclude the possibility of idContainer.id in that throw statement.

Reply:
No, apparently that type check isn't sufficient for `{ id: Id64String }`, only for `RelatedElement`. I can type the argument that way?

Reply:
Im assuming if we ever make it there, we're logging an object which doesnt have an 'id' property defined on it. So not really sure what to do about that... I guess whatever logs the object seems best. I don't know when something gets logged as [Object object] or not though.

Reply:
That's the root of these changes - eslint is complaining that it could be logged `[Object object]`. I'm using `JSON.stringify(idContainer)` now.

Reply:
That seems good to me. Thanks